voltdb/src/main/java/com/yahoo/ycsb/db/package-info.java

```diff
@@ -0,0 +1,7 @@
+/**
+ * VoltDB integration with YCSB.
+ * 
+ * @author srmadscience / VoltDB
+ *
+ */
+package com.yahoo.ycsb.db;
```

Comment:
shouldn't need this package-info file because everything should be in `com.yahoo.ycsb.db.voltdb`

Reply:
Should be fixed in 0a534e8434d7e854901fbcf825df0403cc985ba3 .